src/ray/rpc/grpc_server.cc

```diff
@@ -59,6 +59,13 @@ void GrpcServer::Run() {
                              RayConfig::instance().grpc_keepalive_timeout_ms());
   builder.AddChannelArgument(GRPC_ARG_KEEPALIVE_PERMIT_WITHOUT_CALLS, 0);
 
+  // TODO(rickyyx): This suppresses the gRPC server printing "too many pings" error
+  // message. Suppressing the error should not have any more negative impact on the
+  // system. We will need to further investigate into which set of gRPC settings on
+  // client/server for keepalive are misconfigured:
+  // https://github.com/ray-project/ray/issues/25367
+  builder.AddChannelArgument(GRPC_ARG_HTTP2_MAX_PING_STRIKES, 0);
```

Comment:
default value is 2 w/o this being suppressed. 
We could also do some arbitrary non-zero values, like 10 if suppressing altogether is too much a change.

Reply:
Hmm isn't this causing issues like we cannot properly detect dead components or cannot closing problematic connections?

Reply:
We will still be able to detect dead components since in this case, it is a live client sending too many keep alive pings. 

But now I think about it, I think this might keep MORE connections than before, in a way, more connections will be kept alive. This could actually have an effect on the system. 

Maybe changing the value up more incrementally? e.g. 8?

Reply:
Can you try a couple different values and see when the error disappeared?